imports/plugins/included/ui-search/lib/components/searchModal.js

```diff
@@ -98,6 +98,7 @@ class SearchModal extends Component {
           {this.renderSearchInput()}
           {this.renderSearchTypeToggle()}
           {this.props.tags.length > 0 && this.renderProductSearchTags()}
+          {this.props.value.length >= 3 && this.props.products.length < 1 && <h3><b> No product(s) found </b> </h3>}
```

Comment:
It's great to see you gracefully handled a not found case...

Reply:
Thanks.